recipe/meta.yaml

```diff
@@ -21,8 +21,8 @@ requirements:
     - pip
   run:
     - python
-    - neotime <2,>=1.7.1
-    - neobolt <2,>=1.7.4
+    - neotime <2
+    - neobolt <2
```

Comment:
Why did you revert this change? It is in the package specification/

Reply:
You are rigth, my bad, I accidentally thought that build failed because recipe does not support that, and rerevert it again without noticing, I will revert ot now